docs/source/authentication.rst

```diff
@@ -0,0 +1,19 @@
+Authentication
+--------------
+This section breaks down the different authentication models used within Quipucords from interacting with the server from the command line or how the server interacts with remote systems.
```

Comment:
from the command line or how --> from the command line to how

Reply:
done